README.md

```diff
@@ -325,13 +325,14 @@ Focus is on the specific task.
 *Misc plugins and libraries*
 
 - [Ajax plugin](https://github.com/dereuromark/cakephp-ajax) - A plugin to ease handling AJAX requests.
-- [CakeManager plugin](https://github.com/cakemanager/cakephp-cakemanager) - Manage your app: User-Management, built in admin-area and many components and behaviors available for your app.
+- [CakeManager plugin](https://github.com/cakemanager/cakephp-cakemanager) - Manage your app: User-Management and a built in admin-area. Ready for use :)
```

Comment:
How about:

```
A user management plugin with a built-in admin area.
```

I don't think `Ready for use :)` is necessary as it's the 'awesome' list, so all listed plugins are supposed to be ready for use ;)